scripts/pylib/twister/twisterlib/testinstance.py

```diff
@@ -204,9 +209,16 @@ def setup_handler(self, env):
     # Global testsuite parameters
     def check_runnable(self, enable_slow=False, filter='buildable', fixtures=[], hardware_map=None):
 
-        # running on simulators is currently not supported on Windows
-        if os.name == 'nt' and self.platform.simulation != 'na':
-            return False
+        if os.name == 'nt':
+            # running on simulators is currently supported only for QEMU on Windows
+            if self.platform.simulation not in ('na', 'qemu'):
+                return False
+
+            # check presence of QEMU on Windows
+            if ('\\Program Files\\qemu' not in os.environ['PATH'] and
+                '\\Program Files (x86)\\qemu' not in os.environ['PATH'] and
+                    'QEMU_BIN_PATH' not in os.environ):
```

Comment:
this need be document in commit message at least

Reply:
I'm not using qemu on Windows, but I don't think, that checking `Program Files` as a place, where qemu is installed, is a good method. User probable can select another folder. Maybe checking `QEMU_BIN_PATH` is enough.

Reply:
yes, please remove the hardcoded windows path.

Reply:
Some kind of check of QEMU installation was requested by @PerMac, because qemu tests would start failing instead of being skipped if QEMU is not installed. I can see the below options, but non of them is perfect:

1. Remove the check completely (failed instead of skipped issue would persist)
2. Check if QEMU is installed using Windows registry. It is independent from installation path. Relies only on path of QEMU entry in registry which is given by installer and cannot be changed by a user. This solves skipped vs. failed issue, if QEMU is not installed, but if it is installed, it is not sure, it is actually runnable by Twister, because it needs to be added to PATH or QEMU_BIN_PATH manually.
3. Only check that QEMU_BIN_PATH is in environment variable.  This does not ensure, that QEMU is actually installed. Would probably also require documentation change because documentation would suggest to add QEMU installation path to PATH (#67247) not to QEMU_BIN_PATH (QEMU_BIN_PATH is intended for custom QEMU installation as mentioned in the doc).
4. Combination of 2. and 3. with a check that registry entry and QEMU_BIN_PATH are equal.

Any comments or preference, please?

Reply:
I propose to:
- leave only the check for QEMU_BIN_PATH
- add a statement in docs PR (#67247) that QEMU_BIN_PATH has to be added to the system's variables

Reply:
Done.